### ISSUE TITLE
SSH not responding

### ISSUE THREAD

**Issue by hdprice** [andrewjfreyer/monitor#394]
After about a day of running the Pi seems to stop responding to incoming SSH connections. They fail with a connection timeout error. SSH works OK again after cycling the power. Is anybody else seeing this? Is there any way to troubleshoot it?

**Comment by mzaferyahsi** [comment]
I also noticed the same issue and I found a workaround by running a cron job that reboots the pi periodically :(

**Comment by hdprice** [comment]
> I also noticed the same issue and I found a workaround by running a cron job that reboots the pi periodically :(

Good idea! I've noticed that Monitor stops sending MQTT messages as well. Haven't been able to determine if these events occur at the same time. Rebooting seems to be the only way to get things working again but it's not ideal. I'm hoping @andrewjfreyer will see this and share his thoughts.

**Comment by turboc1208** [comment]
Any update on this?  I've also noticed when the pi reboots or anytime I restart monitor there are about a doze copies of monitor.sh running based on a ps -ef | grep monitor
Thanks
